packages/components/src/image/src/SvgImage.tsx

```diff
@@ -11,28 +11,28 @@ export interface InnerSvgImageProps extends
     Omit<InternalProps, "as">,
     Omit<SVGProps<SVGSVGElement>, keyof StyledSystemProps | OverlappingStyleProps> {
     /**
-     * See [WCAG](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html).
-     */
+         * See [WCAG](https://www.w3.org/TR/WCAG20-TECHS/ARIA14.html).
+         */
     "aria-label": string;
     /**
-     * See [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill).
-     */
+         * See [MDN](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/fill).
+         */
     fill?: FillProp;
     /**
-    * The image height.
-    */
+        * The image height.
+        */
```

Comment:
Not sure why all these comments doesn't follow the same spacing as the other comments